rollup.config.js

```diff
@@ -55,29 +55,8 @@ const config = [
     },
     plugins: [
       ...plugins,
-      babel({
-        // The 'external-helpers' Babel plugin allows Rollup to include every
-        // used babel helper just once per bundle, rather than including them in
-        // every module that uses them (which is the default behaviour).
-        plugins: ['external-helpers'],
-        presets: [
-          ['env', {
-            // Run `npm run browserslist` to see the percent of users covered
-            // by this browsers selection
-            targets: {
-              browsers: pkg.browserslist
-            },
-
-            // Instructs Babel to not convert ES modules to CommonJS--that's a
-            // job for Rollup.
-            modules: false,
-
-            // To see which browsers are targeted, and which JS features are
-            // polyfilled, uncomment the next line and run `npm run build`
-            // debug: true,
-          }]
-        ],
-      })]
+      babel()
```

Comment:
It does not look like with this configuration Babel uses the browsers list from `package.json`. The resulting UMD bundle is not transpiled.

Try building the UMD bundle with `npm run build` before and after this PR and compare the `dist/vaadin-router.umd.js` files. For example the string literals remain, `const` and `let` remain, etc. This needs to be fixed.
This needs to be fixed.